server/utils/nunjucksSetup.ts

```diff
@@ -202,5 +202,19 @@ export function registerNunjucks(app?: express.Express): Environment {
       : undefined
   })
 
+  njkEnv.addFilter('extractAttr', (array, key) => {
```

Comment:
is this actually being used?

Reply:
I was trying it out but then didn't end up using it. Forgot to remove it afterwards.